packages/ng/libraries/formly/src/lib/formly.config.ts

```diff
@@ -43,6 +47,16 @@ export const LU_FORMLY_CONFIG = {
 			component: LuFormlyFieldInput,
 			wrappers: ['textfield-layout'],
 		},
+		{
+			name: 'radio',
+			component: LuFormlyFieldRadios,
+			wrappers: ['textfield-layout'],
+		},
+		{
+			name: 'checkbox',
+			component: LuFormlyFieldCheckboxes,
+			wrappers: ['textfield-layout'],
+		},

```

Comment:
i thought we introduced the notion of `textfield` insted of `field` cuz _standard_ inputs (text, select, textarea) dont behave like checkboxes and radios

Reply:
@jeremie-lucca : Lucien is raising a good point. Should I rename the whole `textfield` into `form-field` ?